ci/apache/run_integration_tests.sh

```diff
@@ -11,6 +11,7 @@ cd $TRAVIS_BUILD_DIR
 
 # Test commands
 py.test --cov=hil --cov-append tests/integration/cli.py
+py.test --cov=hil --cov-append tests/integration/client.py
```

Comment:
this should have taken care of coverage for the client lib tests. Don't know what's wrong with it :/ will have to think about it.